docs/docs/rest-api/public/api/v2/types/healthCheck.raml

```diff
@@ -0,0 +1,33 @@
+#%RAML 1.0 DataType
+type: object
+usage: Must specify a single type of check, http, tcp, or command
+maxProperties: 1
+minProperties: 1
+properties:
+  http?:
+    type: object
+    properties:
+      endpoint:
+        type: !include name.raml
+        description:|
+          The endpoint name to use.
+          In "host" mode health checks use the hostPort. In other modes use the containerPort.
+          If an integer is specified then it should match hostPort or containerPort when in "host"
+      path?:
+        type: string
+        minLength: 1
+        maxLength: 1024
+      scheme?:
+        type: string
+        enum: [ HTTP, HTTPS ]
+        description: The http scheme to use
+  tcp?:
+    type: object
+    properties:
+      type: !include name.raml
```

Comment:
Whoops. Should be name (and you can inline the type of you don't add anything else)